plugins/woocommerce-admin/client/marketplace/components/search-results/search-results.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+import './search-results.scss';
+import { Product, ProductType } from '../product-list/types';
+import Extensions from '../extensions/extensions';
+import Themes from '../themes/themes';
+import { MARKETPLACE_SEARCH_RESULTS_PER_PAGE } from '../constants';
+
+export interface SearchResultProps {
+	products?: Product[];
```

Comment:
Do we need `?` here? I think we may not need to. I traced this prop and it defaults to an empty array. Did a quick test without the `?` and all worked normally. 

Since we made this nullable here, all of the downstream code (Themes, Extensions components) also has to accommodate the possible null value, even though the `products` never go null.

Reply:
It wasn't needed, updated it in [4ea54be](https://github.com/woocommerce/woocommerce/pull/40303/commits/4ea54be3b2a1861a3079aa3081b1ebf9fe8781c2)